python/lsst/pipe/tasks/imageDifference.py

```diff
@@ -603,6 +622,18 @@ def runDataRef(self, sensorRef, templateIdList=None):
                     else:
                         self.measurement.run(diaSources, subtractedExposure, exposure)
 
+            if self.config.doForcedMeasurement:
+                # Run forced psf photometry on the PVI at the diaSource locations.
+                # Copy the measured flux and error into the diaSource.
+                forcedSources = self.forcedMeasurement.generateMeasCat(
+                    exposure, diaSources, subtractedExposure.getWcs())
+                self.forcedMeasurement.run(forcedSources, exposure, diaSources, subtractedExposure.getWcs())
+                mapper = afwTable.SchemaMapper(forcedSources.getSchema(), diaSources.getSchema())
```

Comment:
If you would pythonify this line a little bit by using `forcedSources.schema` and the same for diaSources that would be nice. If it is a big inconvenience it is not strictly necessary, but would make things look a bit nicer and more consistent.